src/WebJobs.Script/Description/DotNet/Compilation/CSharp/Analyzers/AsyncVoidAnalyzer.cs

```diff
@@ -27,6 +27,8 @@ public AsyncVoidAnalyzer()
 
         public override void Initialize(AnalysisContext context)
         {
+            context.EnableConcurrentExecution();
+            context.ConfigureGeneratedCodeAnalysis(GeneratedCodeAnalysisFlags.Analyze | GeneratedCodeAnalysisFlags.ReportDiagnostics);
             context.RegisterSymbolAction(AnalyzeSymbol, SymbolKind.Method);
```

Comment:
Any risk with these changes? Looks like a little more than just a nuget update...

Reply:
These are the same changes I had to make in the 3.0 branch when taking in the new major versions.

Reply:
Oh was that already done and released? Didn't realize this was related. If so, looks good.